cypress/e2e/checkout_spec.cy.js

```diff
@@ -74,4 +75,15 @@ describe('checkout', () => {
       .wait('@createOrderForm')
       .get('.checkout__form__fail').contains('p','Your order request could not be processed at this time. Please try again later.')
     })
+  it('the user should have same items in cart on refresh', () => {
+    cy.reload()
```

Comment:
The network request is stubbed but if it doesn't break, could you add `cy.wait("@GetAllItems")` here?